clientSrc/components/Login.js

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
I would avoid ignoring things like this. If it's not used it's better to remove.